pdf-service/app/main.py

```
@@ -168,6 +170,16 @@ async def _docx_to_pdf_word_first(
         _renderer_path.set("gotenberg_only")
         return await docx_to_pdf(docx_bytes, filename=filename)
 
+    # R42-4 cache lookup（只 cache daemon 路徑；fallback 路徑 fidelity 不同）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)